### DISCUSSION TITLE
Report fraud

### DISCUSSION THREAD

**Discussion by qbuffex** [community/community#17129] [Other Feature Feedback, Questions, & Ideas]
Hello, dear management friend. I'm here to report a liar's account. https://github.com/trewisscotch  https://twitter.com/TrewisScotch   He publishes a lot of code on GIT. It also shows that he is selling these products. He used git to disguise himself, and then many people think that our friends on git are simple and don't cheat. As a result, I cheated him $300 yesterday. Later, I found that many people cheated him and hoped that the management would close his git account. Reduce the chances of others being cheated by him. thank you. He'll go to hell. God must punish him

**Comment by qbuffex** [comment]
![image](https://user-images.githubusercontent.com/98819492/169712059-25b00f18-8fc0-4ee4-8c42-0a1e4ac5e121.png)

**Reply by qbuffex** [reply]
![image](https://user-images.githubusercontent.com/98819492/169712737-06cd308c-0faf-405d-b3a0-20655d78d2f1.png)

**Comment by qbuffex** [comment]
![image](https://user-images.githubusercontent.com/98819492/169712150-03876021-1c16-4ffe-a3be-6ec94ffb0656.png)

**Comment by qbuffex** [comment]
![image](https://user-images.githubusercontent.com/98819492/169712183-90a3c1f5-c1ca-4485-be93-0e596734695b.png)

**Comment by qbuffex** [comment]
![image](https://user-images.githubusercontent.com/98819492/169712312-1da62fe1-3c85-44e7-8993-997841410cc4.png)

**Reply by qbuffex** [reply]
https://github.com/trewisscotch-is-scammer/Cobalt-Strike-License

**Comment by qbuffex** [comment]
The chat record between me and him was blacked out by him. This is the evidence that I sorted out that others were cheated by him. I believe that justice will never be defeated by evil. We should try to reduce the probability of others being cheated. We hope to manage our friends to deal with his account, because 80% of the reason we trust him is his git account. Please deal with this matter carefully. May the world be fair, thank you

**Reply by qbuffex** [reply]
God will not leave any good man, nor will he let go of any bad man

**Reply by edris9274** [reply]
https://github.com/SuperAdam47
This guy is also a scammer ; scammed me 150$
